core/client/fs/src/main/java/alluxio/client/block/stream/GrpcBlockingStream.java

```diff
@@ -70,9 +71,20 @@
    */
   public GrpcBlockingStream(Function<StreamObserver<ResT>, StreamObserver<ReqT>> rpcFunc,
       int bufferSize) {
+    this(rpcFunc, bufferSize, "");
```

Comment:
can we require all streams be created with a description?

Reply:
I will remove this constructor.